pkg/report/report.go

```diff
@@ -521,6 +517,72 @@ func Generate(ctx context.Context, path string, mrs yara.MatchRules, c malconten
 		// TODO: If we match multiple rules within a single namespace, merge matchstrings
 	}
 
```

Comment:
Can some of the override handling code move to a new function? This one is getting overly large and complicated.

Reply:
Moved to a new function in [`660024a` (#487)](https://github.com/chainguard-dev/malcontent/pull/487/commits/660024ab189cc7f5e1bc854b07ddd0c7d50510ba).